File: fix_bracket.js
const fs = require('fs');

function count(str, tag) {
    return str.split(tag).length - 1;
}

const content = fs.readFileSync('src/components/AdminPanel.tsx', 'utf8');
const numOpen = count(content, '<div');
const numClose = count(content, '</div');
console.log('AdminPanel div open:', numOpen, 'close:', numClose);

File: fix_bracket.py
with open('src/components/AdminPanel.tsx', 'r') as f:
    content = f.read()

num_open = content.count('<div')
num_close = content.count('</div')
print('AdminPanel <div:', num_open, '</div:', num_close)

num_open_brace = content.count('{')
num_close_brace = content.count('}')
print('AdminPanel {:', num_open_brace, '}:', num_close_brace)

num_open_paren = content.count('(')
num_close_paren = content.count(')')
print('AdminPanel (:', num_open_paren, '):', num_close_paren)
